manifests/latest/vsphere-7.0u1/vanilla/deploy/validatingwebhook.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: sc-validation-webhook-svc
+  namespace: kube-system
+  labels:
+    app: vsphere-csi-controller
+spec:
+  ports:
+    - port: 443
+      targetPort: 8443
+  selector:
+    app: vsphere-csi-controller
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: validation.csi.vsphere.vmware.com
+webhooks:
+  - name: scvalidation.csi.vsphere.vmware.com
+    clientConfig:
+      service:
+        name: sc-validation-webhook-svc
+        namespace: kube-system
+        path: "/validate-storageclass"
```

Comment:
can we keep a generic name for path and we can have multiple rules if needed here.

Reply:
used validate as the path to help validate all diff types of k8s resources.

```
    clientConfig:
      service:
        name: validation-webhook-svc
        namespace: kube-system
        path: "/validate"
```